configuration/config-txt/video.md

```diff
@@ -426,11 +393,19 @@ The options are:
  - `2` = `EDID_ContentType_Photo`, content type photo
  - `3` = `EDID_ContentType_Cinema`,  content type cinema
  - `4` = `EDID_ContentType_Game`,  content type game
- 
-### hdmi_enable_4kp60 (Pi 4B only)
 
-By default, when connected to a 4K monitor, the Raspberry Pi 4B will select a 30hz refresh rate. Use this option to allow selection of 60Hz refresh rates. Note, this will increase power consumption and increase the temperature of the Raspberry Pi. It is not possible to output 4Kp60 on both micro HDMI ports simultaneously.
- 
+<a name="hdmienable4kp60"></a>
+### hdmi_enable_4kp60 (Pi 4 only)
+
+By default, when connected to a 4K monitor, the Raspberry Pi 4 will select a 30hz refresh rate. Use this option to allow selection of 60Hz refresh rates:
+
+```
+hdmi_enable_4kp60=1
+```
+
+Note: this will increase power consumption and increase the temperature of the Raspberry Pi. It is only possible to output 4Kp60 on HDMI 0.
```

Comment:
ping @JamesH65 for clarification - the previous text implied that you could display 4Kp60 on either HDMI 0 _or_ HDMI 1, but the new text explicitly states 4Kp60 is only possible on HDMI 0 ?

Reply:
> ping @JamesH65 for clarification - the previous text implied that you could display 4Kp60 on either HDMI 0 _or_ HDMI 1, but the new text explicitly states 4Kp60 is only possible on HDMI 0 ?

The old text was wrong - 4K modes at 50 and 60Hz are only available on HDMI 0.

Reply:
Correct.